website/docs/Getting-Started.md

```diff
@@ -2,7 +2,7 @@
 
 <!-- ### Welcome to AutoGen, a library for enabling Next-Gen LLM Applications via Multi-Agent Conversation Framework! -->
 
-AutoGen is a framework that enables development of LLM applications using multiple agents that can converse with each other to solve tasks. AutoGen agents are customizable, conversable, and seamlessly allow human participation. They can operate in various modes that employ combinations of LLMs, human inputs, and tools.
+AutoGen is a programming framework for agentic AI, just like PyTorch for deep learning. It enables development of LLM applications using multiple agents that can converse with each other to solve tasks. AutoGen agents are customizable, conversable, and seamlessly allow human participation. They can operate in various modes that employ combinations of LLMs, human inputs, and tools.
```

Comment:
```suggestion
AutoGen is a programming framework for agentic AI, like PyTorch for deep learning. It enables development of LLM applications using multiple agents that can converse with each other to solve tasks. AutoGen agents are customizable, conversable, and seamlessly allow human participation. They can operate in various modes that employ combinations of LLMs, human inputs, and tools.
```

Reply:
Shall we define "agentic AI", referencing the OpenAI's whitepaper? https://cdn.openai.com/papers/practices-for-governing-agentic-ai-systems.pdf